internal/flink/command_statement_list.go

```diff
@@ -35,12 +36,12 @@ func (c *command) statementList(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	computePoolId, err := cmd.Flags().GetString("compute-pool")
-	if err != nil {
-		return err
+	computePool := c.Context.GetCurrentFlinkComputePool()
+	if computePool == "" {
+		return errors.NewErrorWithSuggestions("no compute pool selected", "Select a compute pool with `confluent flink compute-pool use` or `--compute-pool`.")
```

Comment:
These strings seem to be used multiple times, maybe we can put them in errors package.

Reply:
No longer being used!